lib/auth/authclient/api.go

```diff
@@ -949,7 +949,10 @@ type Cache interface {
 	NewWatcher(ctx context.Context, watch types.Watch) (types.Watcher, error)
 
 	// GetReverseTunnels returns  a list of reverse tunnels
-	GetReverseTunnels(ctx context.Context, opts ...services.MarshalOption) ([]types.ReverseTunnel, error)
+	GetReverseTunnels(ctx context.Context) ([]types.ReverseTunnel, error)
```

Comment:
I wasn't too sure on our preference here - so open to your thoughts. I figure that once v18 hits and we can be sure that all remote calls (via client.GetReverseTunnels) are using ListReverseTunnels under the hood, it's then a fairly trivial task to remove GetReverseTunnels. My only reservation in immediately marking GetReverseTunnels as deprecated and telling people to use ListReverseTunnels was the chance that someone may convert a GetReverseTunnels call to directly depend on ListReverseTunnels before it's "safe" to do so. Part of me wonders if potentially I can be less cautious here now with the recent changes to reject old clients/new servers.

Reply:
Ah - ok - in that case, I'll mark it as deprecated. I'll leave removing it for now since I think it'll introduce a bunch more changes across the codebase, and it'll probably be cleaner to remove it when I come back for the rest of the stuff here.